src/core/hyper-ref/index.js

```diff
@@ -305,10 +304,51 @@ export default class HyperRef extends PureComponent<Props, State> {
       });
     }
 
-    return React.createElement(
-      TouchableOpacity,
-      { ...props, ...pressHandlers, accessible: false },
-      children,
+    const style = this.getStyle();
+    const { accessibilityLabel, testID } = createTestProps(this.props.element);
+    const { onLongPress, onPress, onPressIn, onPressOut } = pressHandlers;
+
+    // If element is a <text> nested under another <text>, simply add press events
+    const isNestedUnderText =
+      this.props.element.parentNode?.namespaceURI === Namespaces.HYPERVIEW &&
+      this.props.element.parentNode?.localName === LOCAL_NAME.TEXT;
+
+    if (isNestedUnderText) {
+      const noop = () => {};
+      return (
+        <Text
+          accessibilityLabel={accessibilityLabel}
+          accessible={false}
+          onLongPress={onLongPress}
+          // when no press handler set, we still need an empty handler for pressIn or pressOut handlers to work
```

Comment:
How did you figure out this issue and the one below for `onResponderRelease`/`onResponderRelease`? Is there an SO of GH issue we can link to?

Reply:
I just ran into the bug myself, I can check if there's any document out there for this

Reply:
I haven't found anything of value in GH/SO so far. I might pause this one a little bit and do a bit more testing on it including Android

Reply:
Tested on Android 12, it's working well.